c7n/resources/glue.py

```diff
@@ -272,3 +272,148 @@ def process(self, resources):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('get_ml_transforms', 'Transforms', None)
+        name = 'Name'
+        id = 'TransformId'
+        arn_type = 'mlTransform'
+        universal_taggable = True
+
+    permissions = ('glue:GetMLTransforms',)
+    augment = universal_augment
+
+
+@GlueMLTransform.action_registry.register('delete')
+class DeleteMLTransform(BaseAction):
+
+    schema = type_schema('delete')
+    permissions = ('glue:DeleteMLTransform',)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('glue')
+        for r in resources:
+            try:
+                client.delete_ml_transform(TransformId=r['TransformId'])
+            except client.exceptions.EntityNotFoundException:
+                continue
+
+
+@resources.register('glue-security-configuration')
+class GlueSecurityConfiguration(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('get_security_configurations', 'SecurityConfigurations', None)
+        id = name = 'Name'
+        arn_type = 'securityConfiguration'
+        date = 'CreatedTimeStamp'
+
+    permissions = ('glue:GetSecurityConfigurations',)
+
+
+@GlueSecurityConfiguration.action_registry.register('delete')
+class DeleteSecurityConfiguration(BaseAction):
+
+    schema = type_schema('delete')
+    permissions = ('glue:DeleteSecurityConfiguration',)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('glue')
+        for r in resources:
+            try:
+                client.delete_security_configuration(Name=r['Name'])
+            except client.exceptions.EntityNotFoundException:
+                continue
+
+
+@resources.register('glue-trigger')
+class GlueTrigger(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('get_triggers', 'Triggers', None)
+        id = name = 'Name'
+        arn_type = 'trigger'
+        universal_taggable = True
+
+    permissions = ('glue:GetTriggers',)
+    augment = universal_augment
+
+
+@GlueTrigger.action_registry.register('delete')
+class DeleteTrigger(BaseAction):
+
+    schema = type_schema('delete')
+    permissions = ('glue:DeleteTrigger',)
+
+    def process(self, resources):
+        client = local_session(self.manager.session_factory).client('glue')
+        for r in resources:
+            try:
+                client.delete_trigger(Name=r['Name'])
+            except client.exceptions.EntityNotFoundException:
+                continue
+
+
+@resources.register('glue-workflow')
+class GlueWorkflow(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'glue'
+        enum_spec = ('list_workflows', 'Workflows', None)
+        detail_spec = ('get_workflow', 'Name', None, None)
+        id = name = 'Name'
+        arn_type = 'workflow'
+        universal_taggable = True
```

Comment:
Just a sanity check here, are the workflows taggable? I did not see that in this document

https://docs.aws.amazon.com/glue/latest/dg/monitor-tags.html

Reply:
hmm strange. Boto3 allows for tags for Workflows https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.create_workflow, and I tested locally tagging and filtering for Workflows with the ResourceGroupsTaggingAPI and it seemed to be working

Reply:
Nice! That documentation must be out of date then. Thanks for confirming!

Reply:
universal_taggable = object()